microservices/services/contest-service/src/main/java/org/xcolab/service/contest/domain/contest/ContestDaoImpl.java

```diff
@@ -14,10 +15,21 @@
 import org.xcolab.service.contest.exceptions.NotFoundException;
 import org.xcolab.service.utils.PaginationHelper;
 import org.xcolab.util.SortColumn;
+import org.xcolab.util.enums.activity.ActivityEntryType;
 
 import java.util.List;
 
+import static org.xcolab.model.Tables.ACTIVITY_ENTRY;
+import static org.xcolab.model.Tables.ACTIVITY_SUBSCRIPTION;
+import static org.xcolab.model.Tables.COMMENT;
 import static org.xcolab.model.Tables.CONTEST;
+import static org.xcolab.model.Tables.CONTEST_DISCUSSION;
+import static org.xcolab.model.Tables.CONTEST_PHASE;
+import static org.xcolab.model.Tables.CONTEST_TEAM_MEMBER;
+import static org.xcolab.model.Tables.CONTEST_TRANSLATION;
+import static org.xcolab.model.Tables.PROPOSAL_2_PHASE;
+import static org.xcolab.model.Tables.THREAD;
+import static org.xcolab.service.proposal.domain.proposal.ProposalDaoImpl.deleteOrphanProposals;
```

Comment:
I would not add static imports for more complicated methods such as this one - it hides their true association in the class body.

Reply:
Done.